code/modules/multiz/turf.dm

```diff
@@ -31,9 +22,17 @@
 	icon_state = ""
 	density = 0
 	pathweight = 100000 //Seriously, don't try and path over this one numbnuts
-
 	z_flags = ZM_MIMIC_DEFAULTS | ZM_MIMIC_OVERWRITE | ZM_MIMIC_NO_AO | ZM_ALLOW_ATMOS
 
+/turf/simulated/open/CanZPass(atom/A, direction)
```

Comment:
Not in scope for this PR, but I feel like this behavior should be moved down to `/turf` and gated by a ZM flag or the like at some point.